oioioi/acm/tests.py

```diff
@@ -145,3 +146,26 @@ def test_model_solution_submission_view(self):
     def test_safe_exec_mode(self):
         contest = Contest.objects.get()
         self.assertEqual(contest.controller.get_safe_exec_mode(), 'cpu')
+
+
+class TestACMScores(TestCase):
+    fixtures = ['acm_test_full_contest']
+
+    def _get_badge_for_problem(self, content, problem):
+        soup = bs4.BeautifulSoup(content, 'html.parser')
+        print(content)
```

Comment:
please remove debug print

Reply:
Apparently, I forgot to git add this file... Fixed!